P5/Source/Guidelines/en/ND-NamesDates.xml

```diff
@@ -2269,6 +2085,265 @@ occupation and residence.</p>
       </div>
     </div>
 
+    <div xml:id="NDEVNT" type="div2">
+      <head>Events</head>
+      <p>
+        <specList>
+          <specDesc key="event"/>
+          <specDesc key="eventName"/>
+          <specDesc key="listEvent"/>
+        </specList> An event is a change of state whether real, planned, fictional, or purported about which more
```

Comment:
Council F2F in Paderborn: We can simplify this as simply: Broadly speaking, an event is something that happens, whether real, planned, fictional, or purported.